sdk/ml/azure-ai-ml/tests/schedule/e2etests/test_schedule.py

```diff
@@ -16,6 +16,7 @@
 @pytest.mark.e2etest
 @pytest.mark.usefixtures("recorded_test", "mock_code_hash", "mock_asset_name", "mock_component_hash")
 @pytest.mark.pipeline_test
+@pytest.mark.mock_component_test
```

Comment:
What's this mark used for?

Reply:
This is to rerun related tests, should be removed